ansible/roles/jetson_services/files/rover-jetson-raman.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=MRover Onboard Raman Spectroscope
```

Comment:
so jetson/raman existed in the past? should delete maybe?

Reply:
im deleting the raman folder, we will have a raman service when you guys get the port forwarding working